Array Processing Tool/array_processing_tool.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Array Processing Tool</title>
+        <link rel="stylesheet" href="main.css" type="text/css">
+    </head>
+    <body>
+        <section>
+            <h1>Sub sum</h1>
+            <form id="form_subSum">
+                <span>Enter array: </span>
+                <input id="arr_subSum" type="text" list="arrays">
```

Comment:
![image](https://user-images.githubusercontent.com/2150198/93859539-6cf5e580-fcc6-11ea-83af-3c90e5b93786.png)
looks like not working correctly

Reply:
or it is not pulling the data from the input